main.go

```diff
@@ -22,6 +22,7 @@ import (
 
 	"github.com/ianschenck/envflag"
 	"github.com/linode/linodego"
+	_ "go.uber.org/automaxprocs"
```

Comment:
Could you do something like: https://pkg.go.dev/go.uber.org/automaxprocs/maxprocs#example-Logger

Reason: We want to log what maxproc is being set to and also confirm this automaxproc is working

Reply:
Sure, I can add this and push an update